Src/GoogleApis.Auth/OAuth2/IScopableCredential.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+Copyright 2015 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Google.Apis.Auth.OAuth2.Responses;
+using Google.Apis.Http;
+
+namespace Google.Apis.Auth.OAuth2
+{
+    /// <summary>
+    /// Interface for credential types that support alerting scopes
+    /// </summary>
+    public interface IScopableCredential 
+    {
+        /// <summary>
+        /// Only returns true if this credential type has no scopes by default and requires 
+        /// a call to createScoped before use. 
```

Comment:
link to createScoped.
I would rephrase the comment to start with "Returns".
Returns true only if this credential...